instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/OpenTelemetryConnection.java

```diff
@@ -48,6 +51,8 @@
  */
 public class OpenTelemetryConnection implements Connection {
 
+  private static final boolean TXN_ENABLED =
+      ConfigPropertiesUtil.getBoolean("otel.instrumentation.jdbc.experimental.txn.enabled", false);
```

Comment:
should be configured in `JdbcTelemetryBuilder` using a setter and also in `OpenTelemetryDriver`